package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "pass",
   "homepage": ".",
-  "version": "0.0.5",
+  "version": "0.0.5.1",
```

Comment:
Another naming schema I've seen is `0.0.5+x`, the x being a build number. I think what we have here is fine though.